fog/api/proto/view.proto

```diff
@@ -231,6 +217,8 @@ message FogTxOut {
 /// and that's the version of the TxOut that you should use when building a transaction.
 message TxOutRecord {
     /// The (compressed ristretto) bytes of commitment associated to amount field in the TxOut that was recovered
+    ///
+    /// Note: This field is omitted in recent versions, because it can be reconstructed by the recipient instead.
```

Comment:
You can annotate it as deprecated: see `deprecated` on https://developers.google.com/protocol-buffers/docs/proto3

Reply:
for running deployments of fog, though, we can't updated these bytes. so whether the field can be deprecated depends on if those fogs will be started after beta or something like this. otherwise, sometimes people will have to keep reading this field

Reply:
after further discussion, we cannot deprecate this field, we believe it will continue to exist in the wild